draft-ietf-tls-exported-authenticator.md

```diff
@@ -202,15 +219,17 @@ In addition to "signature_algorithms" and "signature_algorithms_cert",
 the "server_name" {{!RFC6066}}, "certificate_authorities"
 (Section 4.2.4. of {{!TLS13}}), and "oid_filters"
 (Section 4.2.5. of {{!TLS13}}) extensions are used to guide certificate
-selection.  The extensions, or others that might affect certificate selection,
-are taken from the authenticator request if present, or the TLS handshake if not.
+selection.
 
-Alternative certificate formats such as {{!RFC7250}} Raw Public Keys
-are not supported in this version of the specification.
+Only the X509 certificate type defined in {{!TLS13}} is supported.
```

Comment:
X.509, no?

Reply:
Per https://tools.ietf.org/html/rfc8446#section-4.4.2 the description language's element is spelled this way.